src/main/java/com/meilisearch/sdk/Documents.java

```diff
@@ -129,10 +166,12 @@ TaskInfo addDocuments(String uid, String document, String primaryKey)
      */
     TaskInfo updateDocuments(String uid, String document, String primaryKey)
             throws MeilisearchException {
-        String urlPath = "/indexes/" + uid + "/documents";
+        URLBuilder urlb = new URLBuilder();
+        urlb.addSubroute("indexes").addSubroute(uid).addSubroute("documents");
         if (primaryKey != null) {
-            urlPath += "?primaryKey=" + primaryKey;
+            urlb.addParameter("primaryKey", primaryKey);
         }
+        String urlPath = urlb.getURL();
```

Comment:
I think it would be: 
```
    public String toQuery(String uid, String identifier, String primaryKey) {
```
Which I don't think exists yet in the toQuery

Reply:
> I think it would be:
`    public String toQuery(String uid, String identifier, String primaryKey) {`

It could but we don't use `identifier`

Reply:
> @brunoocasali what do you think?
Same